package.json

```diff
@@ -64,9 +64,6 @@
       "verbosity": 1
     }
   },
-  "optionalDependencies": {
```

Comment:
So, if the entirety of `optionalDependencies` has been removed, does that mean the entirety of `shrinkwrapIgnore` should be removed too?